Modules/Core/Common/include/itkImageToImageFilterDetail.h

```diff
@@ -54,7 +54,7 @@ namespace ImageToImageFilterDetail
 struct DispatchBase
 {};
 
-/** \struct BooleanDispatch
+/**
  * \brief Templated class to produce a unique type "true" and "false".
```

Comment:
You might as well merge those two lines of comment (line 57 and 58), once  `\struct BooleanDispatch` is removed. As in:

```
  /** \brief Templated class to produce a unique type "true" and "false".
```

Just a suggestion!

Reply:
Yes, I applied the suggestion. Thank you!